### ISSUE TITLE
Should testing use Specta?

### ISSUE THREAD

**Issue by robb** [robb/Underscore.m#30]
I could imagine this bringing down the line count of the specs.

**Comment by stigi** [comment]
If you're looking for a +1, here it is :+1:

**Comment by robb** [comment]
@vascoorey what's your take on this? It's a pretty dull task but I guess we should bite the bullet at some point

**Comment by vascoorey** [comment]
Will need to happen somewhere along the way.. I'll try to have a look this weekend as this week's looking to be a busy one ;)

**Comment by robb** [comment]
yeah, maybe we just start with the string stuff and take it from there. I'll try to set up specta/expecta sometime this week

**Comment by kvnify** [comment]
I'll start to transition the rest of the tests over the next month or so.
